specs/deneb/beacon-chain.md

```diff
@@ -293,9 +293,13 @@ def verify_and_notify_new_payload(self: ExecutionEngine,
     """
     Return ``True`` if and only if ``new_payload_request`` is valid with respect to ``self.execution_state``.
     """
+    # [Modified in Deneb:EIP4788]
     execution_payload = new_payload_request.execution_payload
     parent_beacon_block_root = new_payload_request.parent_beacon_block_root
```

Comment:
Good catch on the modified comment. Could you put it at the end of the `parent_beacon_block_root` line?
```suggestion
    execution_payload = new_payload_request.execution_payload
    parent_beacon_block_root = new_payload_request.parent_beacon_block_root  # [Modified in Deneb:EIP4788]
```